iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/common/schematree/ISchemaTree.java

```diff
@@ -71,6 +70,20 @@ Pair<List<MeasurementPath>, Integer> searchMeasurementPaths(
 
   void mergeSchemaTree(ISchemaTree schemaTree);
 
+  /**
+   * Check whether this schema tree has normal series(not template series).
+   *
+   * @return true if it has normal series, else false
+   */
+  boolean hasNormalTimeSeries();
```

Comment:
Seems TestOnly?

Reply:
> Seems TestOnly?

Query optimization will use this interface.